src/base/NekRSProblem.C

```diff
@@ -68,6 +68,13 @@ NekRSProblem::validParams()
     "Whether to conserve the heat flux by individual sideset (as opposed to lumping all sidesets "
     "together). Setting this option to true requires syntax changes in the input file to use "
     "vector postprocessors, and places restrictions on how the sidesets are set up.");
+  params.addParam<Real>(
+      "initial_mesh_vel", 0.0, "Initial mesh velocity to pass to NekRS on the first timestep");
```

Comment:
Is there ever a situation where you would set this to something other than zero? Also I am a bit confused on the dimensions here. Is this interpreted as `velocity * hat(n)`?

Reply:
I had added this in at one point for testing purposes, it doesn't get dotted with the normal, no, it just applies to all fields. I can just remove it for this pull request actually

Reply:
Yes, I think let's remove it for simplicity. We can always re-add it if necessary

Reply:
You'll need to remove it as an option here, too - otherwise people may think this does something, when it does not.